### DISCUSSION TITLE
How do I access the liked playlist? Seems like none is generated.

### DISCUSSION THREAD

**Discussion by FFFluoride** [sayaka-sh/spmp#408] [Q&A] [answered]
I liked this song. 
![IMG_20241205_002858](https://github.com/user-attachments/assets/a2f81203-cb3e-4d8c-98bd-aa4edd50db08)

Clicked this icon
![IMG_20241205_002924](https://github.com/user-attachments/assets/24ec72d4-14e6-43a2-85c1-f2877a22b4dd)

But no playlist exists. 
![IMG_20241205_002938](https://github.com/user-attachments/assets/cf91e8e4-9c6d-413f-b4e4-cc8be84b75a2)

**Comment by sayaka-sh** [comment, marked as answer]
You're viewing local playlists, press the cloud icon at the bottom of the page to view remote (account) playlists. Alternatively, go to the songs page in the library and press the heart icon to directly view a list of liked songs.

**Reply by FFFluoride** [reply]
Thanks for responding so quicky.
I don't know where this icon is, could you direct me?

**Reply by sayaka-sh** [reply]
<img src="https://github.com/user-attachments/assets/e50d06fa-533c-461b-9db0-90a742e5cc0f" width=400px>

**Reply by FFFluoride** [reply]
Thank you very much.

**Reply by sayaka-sh** [reply]
No problem :+1:

**Reply by mugehan** [reply]
Follow the action you prompted to prompt this.

**Reply by mugehan** [reply]
![Screenshot_20241205_151700.jpg](https://github.com/user-attachments/assets/c7625661-324f-4192-b22a-ba41d0ad2bcc)

**Reply by mugehan** [reply]
java.io.EOFException: Not enough data available
	at io.ktor.utils.io.ByteChannelCtorKt.readByte(Unknown Source:100)
	at io.ktor.utils.io.ByteReadChannelOperationsKt$readByte$1.invokeSuspend(Unknown Source:10)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:8)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:112)
	at androidx.core.app.ActivityRecreator$1.run(Unknown Source:98)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Unknown Source:2)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(Unknown Source:95)

**Reply by mugehan** [reply]
![Screenshot_20241205_175803.jpg](https://github.com/user-attachments/assets/41967dc0-d89b-4e30-ba54-2cf0d9ffcdbd)
